scripts/links.py

```diff
@@ -22,8 +24,8 @@ def _deny_filter(path):
 def urls(files: Iterable[Path]) -> Iterable[Tuple[Path, str]]:
     """Returns an iterable over all urls contained in the provided files"""
 
-    def should_filter(url):
-        _filtered = []
+    def should_filter(url: str) -> bool:
+        _filtered: Container[str] = []
```

Comment:
same question, as with the Sequence, or is it, that mypy suggest the most abstract type that fits

Reply:
Similar as above, but here the `in` operator is required by the type for the functionality.